autogpt_platform/backend/backend/executor/database.py

```diff
@@ -173,9 +173,6 @@ def get_service_type(cls):
     spend_credits = _(d.spend_credits)
     get_credits = _(d.get_credits)
 
-    # Summary data - async
-    get_user_execution_summary_data = _(d.get_user_execution_summary_data)
```

Comment:
@ntindle seems to be a good change, I think you introduced other unused methods as well?